mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule MixTestWatch.Mixfile do
     [
       app: :mix_test_watch,
       version: @version,
-      elixir: "~> 1.5",
+      elixir: "~> 1.12",
```

Comment:
Elixir 1.12 is the minimum version still supported by OTP 24, see https://hexdocs.pm/elixir/1.18.4/compatibility-and-deprecations.html#between-elixir-and-erlang-otp